app/model.rb

```diff
@@ -17,15 +18,19 @@ class User
 DataMapper.finalize.auto_upgrade!
 
 class Ocr
-  def init
-    @engine = Tesseract::Engine.new {|engine|
-                engine.language = :lol # arbitrary
+  def initialize
+    @engine = ::Tesseract::Engine.new {|engine|
+                engine.language = :eng
                 engine.page_segmentation_mode = 4
-                engine.whitelist = [*'a'..'z', *'A'..'Z', *0..9].join # thanks ruby, made this easy
+                engine.whitelist = [*'a'..'z', *'A'..'Z', *0..9].flatten # thanks ruby, made this easy
```

Comment:
I see. Interesting. Thanks for hte direct reff to tesseract docs I'd been reading the gem's source. Major brainfart there. Makes it a lot clearer. If you couldn't tell from the Python implementation tutorial it was much less robust, so I am diving into this head first and am relatively clueless. 

I set it to 4 because my intention is to start with rows of text, and slow expand, using that as the first API layer we toy with. This is not expressed clearly or well at all thus far. So, knowing that if you get a cool idea of presentation roll with it, else I've something I'm going to get together this weekend along those lines.

This whole thing is quite nebulous at this point as its a 2 day old repository, so radical changes are no worry. Go hog wild.